lib/puppet/configurer.rb

```diff
@@ -3,6 +3,7 @@
 require 'timeout'
 require 'puppet/network/http_pool'
 require 'puppet/util'
+require 'puppet/util/retryaction'
```

Comment:
The PR's been merged, with the file name changed to retry_action as Rob had suggested.